README.md

```diff
@@ -40,7 +40,7 @@
 - [Book a call](#📞-book-a-call)
 
 ## About crowd.dev
-crowd.dev is the developer data platform (DDP) that lets companies centralize all touch points developers have with their product and brand, be it in community (e.g. Stack Overflow or Reddit), product (open-source or SaaS), or commercial channels (e.g. HubSpot). The platform pulls data from a variety of different sources, normalizes it, matches identities across platforms, and enriches it with 3rd party data. The result is a unified 360-view of who the developers are that engage with your product and community, which companies they work for, and where they stand in their personal customer journey. 
+crowd.dev is the developer data platform (DDP) that lets companies centralize all touch points developers have with their product and brand, be it in community (e.g. Stack Overflow or Reddit), product (open-source or SaaS), or commercial channels (e.g. HubSpot). The platform pulls data from a variety of different sources, normalizes it, matches identities across platforms, and enriches it with 3rd party data. The result is a unified 360-degree view of  the developers who engage with your product and community, the companies they work for, and their position  they stand in their personal customer journey. 
```

Comment:
`and their position they stand in their personal custom journey` seems a but confusing.
Could you leave this last past of the sentence as it was?
`where they stand in their personal customer journey`

Reply:
@joanagmaia Thank you for your review. I have made the changes that you have requested. please check.